pytealutils/storage/blob/blob.py

```diff
@@ -0,0 +1,206 @@
+from pyteal import (
+    Extract,
+    GetByte,
+    SetByte,
+    Concat,
+    Substring,
+    Len,
+    Or,
+    Itob,
+    BytesZero,
+    Subroutine,
+    Int,
+    TealType,
+    ScratchVar,
+    For,
+    App,
+    Expr,
+    Seq,
+    Bytes,
+    If,
+)
+
+max_keys = 16
+page_size = 128 - 1  # need 1 byte for key
+max_bytes = max_keys * page_size
+max_bits = max_bytes * 8
+
+maxKeys = Int(max_keys)
+pageSize = Int(page_size)
+maxBytes = int(max_bytes)
```

Comment:
It looks like you're using the convention that `camelCase` is a PyTEAL type while `snake_case` is python. That's fine, but here `maxBytes` is just a simple int (and identical to `max_bytes`). So I recommend dropping `maxBytes` completely.

Reply:
oops thats suppoed to be Int, The camel case is just a way to discern between snake case ints and wrapped Ints here, maybe I'll make the above ones underscore prefixed